contracts/GovHub.template

```diff
@@ -166,7 +167,7 @@ contract GovHub is System, IParamSubscriber {
       );
     }
 
-    uint256 startBlock = block.number + 1;
+    uint256 startBlock = block.number + 1; //@openissue
```

Comment:
voting period is meant for humans so why not use human time i.e. block.timestamp?
since the time factors here are way larger than 10 seconds node manipulation is a non-issue
and doing so will provide a simpler human interface by omitting the need for the proposer to convert human time to blocks

Reply:
The only benefit I can think of for using a block.number is the fact that it cannot be manipulated by miners, unlike timestamp that can be manipulated by a bad miner but only for values lesser than 12 seconds. 

But this comes with a price namely that the larger the block.number interval is, the harder it is to accurately convert it to clock time - it sort of shifts away.

Since the time intervals we're dealing with here are way larger than 12 seconds I believe the latter outweighs the prior.

Still I realize that we're on a working platform and far from the design board so I assume we'll settle with what we have.